apps/settings/js/modules/date_time.js

```diff
@@ -271,7 +265,7 @@ define(function(require) {
     var d = new Date();
     this._time = this._formatTime(d);
 
-    var remainMillisecond = (59 - d.getSeconds()) * 1000;
+    var remainMillisecond = (60 - d.getSeconds()) * 1000;
```

Comment:
I honestly don't know why this is 59. This PR in 2012 first introduced this number:
https://github.com/mozilla-b2g/gaia/commit/9707916c6832b508a161595b42221b5bd4611359#diff-c4cdc43d1ddef35f073dece324b4c45dR72
I guess this was copied by Clock app, which also had a similiar logic:
https://github.com/mozilla-b2g/gaia/pull/1827/files#diff-5a237cd9d755aa01f9c7b4b0bddd2d6eR39

so maybe @timdream knows the story.